content/en/opentelemetry/troubleshooting.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Troubleshooting
+kind: documentation
+further_reading:
+- link: "https://opentelemetry.io/docs/collector/troubleshooting/"
+  tag: "OpenTelemetry"
+  text: "OpenTelemetry Troubleshooting"
+---
+
+If you experience unexpected behavior using OpenTelemetry with Datadog, this guide may help you resolve the issue. If you continue to have trouble, contact [Datadog Support][1] for further assistance.
+
+## Different Kubernetes hostname and node name
+
+When deploying in Kubernetes, if the hostname reported by Datadog does not match the expected node name, this is typically the result of missing `k8s.node.name` (and optionally `k8s.cluster.name`) tags.
+
+To troubleshoot, make sure your application deployment and Collector are configured correctly.
+
+Configure the `k8s.pod.ip` attribute for your application deployment: 
+
+```yaml
+env:
+  - name: MY_POD_IP
+    valueFrom:
+      fieldRef:
+        apiVersion: v1
+        fieldPath: status.podIP
+  - name: OTEL_RESOURCE
+    value: k8s.pod.ip=$(MY_POD_IP)
+```
+
+Enable the `k8sattributes` processor in your Collector:
+
+```yaml
+k8sattributes:
+[...]
+processors:
+  - k8sattributes
+```
+
+## Unexpected hostnames with AWS Fargate deployment
+
+In AWS Fargate environments, an incorrect hostname might be reported for traces.
+
+To troubleshoot, make sure to use the `resourcedetection` processor in your Datadog exporter configuration and enable the `ecs` detector.
+
+```yaml
+processors:
+  resourcedetection:
+    detectors: [env, ecs]
+    timeout: 2s
+    override: false
+```
+
+**Note**: ECS Fargate is supported for v0.42+ of the Datadog Exporter.
```

Comment:
What does that mean?
The `processor` is a different component of the collector.
It adds resource attributes to the data which will be sent (traces, metrics and logs), depending on how you configure your pipelines.

How come that be supported just for `v0.42+` of the Datadog Exporter, which is a different component of the collector?
Are we dropping the resource attributes if it is an early version?

Reply:
Thanks! I'll make an adjustment here. I didn't mean to say Exporter configuration.

I'm not 100% sure about the Exporter version. I know that support for Fargate was added here: https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/6670. This was something I grabbed from Support. Maybe I'll remove the note for now as it may just add more noise to the section.